Backend/src/services/TodoServices.ts

```diff
@@ -0,0 +1,50 @@
+import { TodoDbManager } from "../DbManager/TodoDbManager"; 
```

Comment:
For service file can we have naming in format <service-name>.service.ts
Example **todo.service.ts**
Same please check for db-manager and controller